src/NServiceBus.AzureStoragePersistence/SagaPersisters/Azure/AzureSagaPersister.cs

```diff
@@ -159,7 +156,11 @@ public AzureSagaPersister(string connectionString, bool autoUpdateSchema)
 
         void AddObjectToBatch(TableBatchOperation batch, object entity, string partitionKey, string rowkey = "")
         {
-            if (rowkey == "") rowkey = partitionKey; // just to be backward compat with original implementation
+            if (rowkey == "")
```

Comment:
:+1: for having it on a new line _and_ in a block. Easier to step through and see what's executed.